receiver/zookeeperreceiver/metadata.yaml

```diff
@@ -3,11 +3,11 @@ type: zookeeper
 status:
   class: receiver
   stability:
-    unmaintained: [metrics]
+    alpha: [metrics]
   distributions: [contrib]
   codeowners:
+    active: [akats7]
     emeritus: [djaglowski]
-    seeking_new: true
```

Comment:
You're welcome to leave this as true, if you're ever interested in having others contribute more here, but your call.